types/node/ts3.2/node-tests.ts

```diff
@@ -49,6 +49,8 @@ import { BigIntStats, statSync, Stats } from 'fs';
         // $ExpectType number
         const b = value;
     }
+
+    const arg1UnknownErrorA: (arg: string) => Promise<number> = promisify((arg: string, cb: (err: unknown, result: number) => void): void => { });
```

Comment:
We should also fix the same problem for other usages of `PoorMansUnknown`. For example, this currently fails:

```ts
/*
Argument of type '{ [key: string]: unknown; }' is not assignable to parameter of type 'ParsedUrlQueryInput'.
  Index signatures are incompatible.
    Type 'unknown' is not assignable to type '{} | null | undefined'.
      Type 'unknown' is not assignable to type '{}'.
*/
const unknownObj: { [key: string]: unknown; } = {};
querystring.stringify(unknownObj);
```

It would be better if we could override `PoorMansUnknown` to make it actually `unknown`, but I don't think that's possible.